packages/react-composites/playwright.config.ts

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { PlaywrightTestConfig } from '@playwright/test';
+const config: PlaywrightTestConfig = {
+  timeout: 120000, // Allow for more than the standard 30000. This ensures we get debug stack traces when await functions timeout.
+  workers: 1, // Ensure tests run sequentially.
```

Comment:
It's great to set `workers` here, since all hell will break loose if we run concurrently.

Please expand on the comment to say why we need to run these tests sequentially.

Reply:
Updated 👍